python/graphstorm/dataloading/dataloading.py

```diff
@@ -991,6 +991,8 @@ def __init__(self, dataset, target_idx, batch_size, num_negative_edges, fanout=N
                                 "is %d, which is smaller than the expected"
                                 "test size %d, force it to %d",
                                 etype, len(t_idx), self._fixed_test_size[etype], len(t_idx))
+                self._fixed_test_size[etype] = len(t_idx)
```

Comment:
If the `t_idx` is empty, will `len(t_idx)` be 0?

Reply:
t_idx will be an empty tensor. and len(t_idx) will be 0.